examples/pojo-jar/README.md

```diff
@@ -0,0 +1,48 @@
+# camel-k-jar
+This repository takes the example from here https://camel.apache.org/manual/rest-dsl.html#_openapi_swagger_api
```

Comment:
I'd suggest to replace `This repository` with something like `This example ports the following examples to Camel K`.